Atomic/Export.py

```diff
@@ -38,7 +49,7 @@ def export_docker(graph, export_location):
         #export docker volumes
         export_volumes(graph, export_location)
     except:
-        error = sys.exc_info()[0]
+        error = sys.exc_info()[1]
```

Comment:
sys.exc_info is a 3 tuple structure {type, value, traceback}. Since earlier I was reading index 0 (type) it was not printing the error message. sys.exc_info()[1] will print the correct error message. This should fix all our error handling problems.